src/git_changelog/cli.py

```diff
@@ -458,11 +484,19 @@ def parse_settings(args: list[str] | None = None) -> dict:
     elif str(config_file).strip().lower() in ("yes", "default", "on", "true", "1"):
         config_file = DEFAULT_CONFIG_FILES
 
+    jinja_context = explicit_opts_dict.pop("jinja_context", {})
+
     settings = read_config(config_file)
 
     # CLI arguments override the config file settings
     settings.update(explicit_opts_dict)
 
+    # Merge jinja context, CLI values override config file values.
+    if jinja_context and settings.get("jinja_context"):
+        settings["jinja_context"].update(jinja_context)
+    elif jinja_context:
+        settings["jinja_context"] = jinja_context
```

Comment:
IIUC, `settings["jinja_context"]` is always a dict, so we can safely update it directly.

```suggestion
    settings["jinja_context"].update(jinja_context)
```